packages/angular/projects/ui-angular/src/lib/primitives/password-field/password-field.component.html

```diff
@@ -1,58 +1,85 @@
-<label
-  class="amplify-label"
-  [class.amplify-visually-hidden]="labelHidden"
-  [for]="fieldId"
+<div
```

Comment:
Git is showing more line changes than this actually is due to spacing changes; I just changed classnames and aligned html structure with React one.